src/add-api/index.md

```diff
@@ -0,0 +1,31 @@
+# How do I add scoreboards and trophies to my game?
+
+First, you need to implement the Game API in your game's code. Then, you can add scoreboards and trophies to your game.
+
+## Scoreboards
```

Comment:
Clarify that guest scoring is the ability for anyone to submit a score without requirement for a Game Jolt account. Guest scoring should be discouraged, but not too harshly. It has potential for raunchy names, whereas with game jolt usernames we tend to moderate them. But we allow it so they have freedom to be apart from the game jolt user system if they'd like. 

Unique scoring. Having it disabled will show all of a users scores. So if they have the 1st best score and 2nd best score, it will show them in both of those spaces. If they have unique scoring on, then it will only show their best score, and the 3rd best will be in the 2nd spot. If they have guest scoring turned on, and unique scoring also turned on, it will show scores based on distinct guest name. Dad is different than DAD for guest unique scoring. It's case sensitive.

Mention multiple scoreboards. Give examples. "fastest playtime for each level", "global kill count", "highest level attained", etc...

Explain the difference between score strings (the text that shows for a user's score), and the sort value (the number that we sort by). We have them different so that they can format the score such as "6hrs, 5mins, 2secs" but have the sort value be the number of total seconds so we know what rank to give that score.

Explain and give examples of how to use extra data. It is extra information for the developer to know if the user cheated to get that score. They should store information such as playtime while getting the score, number of keypress, etc? Maybe @YLivay would have good helpful suggestions for this. We should mention that the user never sees it. It's only accessible from the developer's dashboard.